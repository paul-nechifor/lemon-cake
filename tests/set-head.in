(= l :(1 2 3))
(set-head l 5)
l
(set-head (tail l) 6)
l
