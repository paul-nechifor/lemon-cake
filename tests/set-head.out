(1 2 3)
5
(5 2 3)
6
(5 6 3)
